charts/consul/test/unit/client-daemonset.bats

```diff
@@ -1241,24 +1241,36 @@ load _helpers
       yq -r '.command | any(contains("consul-k8s-control-plane acl-init"))' | tee /dev/stderr)
   [ "${actual}" = "true" ]
 
-local actual=$(echo $object |
+  local actual=$(echo $object |
       yq -r '.command | any(contains("secret-name"))' | tee /dev/stderr)
-  [ "${actual}" = "false" ]  
+  [ "${actual}" = "false" ]
 
   local actual=$(echo $object |
-      yq -r '.command | any(contains("k8s-namespace"))' | tee /dev/stderr)
-  [ "${actual}" = "false" ]  
+      yq -r '.env[1].name | contains("CONSUL_ADDRESSES")' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
 
   local actual=$(echo $object |
-      yq -r '.command | any(contains("component-name=client"))' | tee /dev/stderr)
-  [ "${actual}" = "true" ]  
+      yq -r '.env[1].value | contains("release-name-consul-server.default.svc")' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
 
   local actual=$(echo $object |
-      yq -r '.command | any(contains("init-type=\"client\""))' | tee /dev/stderr)
-  [ "${actual}" = "true" ]  
+      yq -r '.env[6].name | contains("CONSUL_LOGIN_AUTH_METHOD")' | tee /dev/stderr)
```

Comment:
To make these environment variable tests less flaky, could you use $env instead of $object to get the variables?

See: https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/test/unit/sync-catalog-deployment.bats#L73